README.md

```diff
@@ -45,7 +45,7 @@ You can go to this [repo](https://github.com/pypa/pipenv) for detail information
                                                            |
                  +-------------------------------+         |
                  |       graph transformer       |         |
-                 | (legalization & optimization) | <------‘ 
+                 | (legalization & optimization) | <------/ 
```

Comment:
I prefer original `‘` XDDD

Reply:
The original `'` is a non standard unicode char and was breaking CI for a very silly reason. That's the only reason I changed it :)